script/deploy-canary-demo

```diff
@@ -21,5 +25,8 @@ npm run deploy -- -b chime-sdk-meeting-readiness-checker-gamma-canary -s chime-s
 echo "Deploying to beta stage for canary that talks to gamma Chime endpoint"
 export AWS_ACCESS_KEY_ID=$BETA_AWS_ACCESS_KEY_ID
 export AWS_SECRET_ACCESS_KEY=$BETA_AWS_SECRET_ACCESS_KEY
-npm run deploy -- -b chime-sdk-demo-beta-canary -s chime-sdk-demo-beta-canary -o chime-sdk-demo-beta-canary -i eu-south-1 -p $GAMMA_CHIME_SERVICE_PRINCIPAL -c $GAMMA_CHIME_ENDPOINT -t -l
+npm run deploy -- -b chime-sdk-demo-beta-canary -s chime-sdk-demo-beta-canary -o chime-sdk-demo-beta-canary -i eu-south-1 -p $GAMMA_CHIME_SERVICE_PRINCIPAL -c $GAMMA_CHIME_ENDPOINT -u false -t -l
+npm run deploy -- -r us-east-1 -b chime-sdk-demo-beta-canary-iad -s chime-sdk-demo-beta-canary-iad -o chime-sdk-demo-beta-canary-iad -p $GAMMA_CHIME_SERVICE_PRINCIPAL -c $GAMMA_CHIME_ENDPOINT_IAD -t -l
```

Comment:
Same applies for not using the region names like IAD, FRA outside like this so lets remove it.